be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/handlers/ElectionHandler.scala

```diff
@@ -51,9 +51,8 @@ class ElectionHandler(dbRef: => AskableActorRef) extends MessageHandler {
       electionId: Hash = data.id
       electionChannel: Channel = Channel(s"${rpcMessage.getParamsChannel.channel}${Channel.CHANNEL_SEPARATOR}$electionId")
       keyPair = KeyPair()
-      _ <- dbActor ? DbActor.WriteAndPropagate(rpcMessage.getParamsChannel, message)
       _ <- dbActor ? DbActor.CreateChannel(electionChannel, ObjectType.ELECTION)
```

Comment:
Is this `CreateChannel` necessary ? Since, it seems that you use `createChannel` function inside the `WriteSetupElectionMessage`

Reply:
Good point ! Fixed.